fossevents/events/forms.py

```diff
@@ -0,0 +1,33 @@
+from django import forms
+
+from .models import Event
+
+
+class EventForm(forms.ModelForm):
+    description = forms.CharField(widget=forms.Textarea(attrs={'placeholder': 'Enter description here Markdown'}),)
```

Comment:
Copy update: `Enter description...`
And you can have `Markdown supported` as help text with a link to simplified markdown help page.